models/timeswap/arbitrum/timeswap_arbitrum_lend.sql

```diff
@@ -0,0 +1,138 @@
+{{ config(
+    alias = 'lend',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['transaction_hash', 'pool_pair', 'maturity', 'strike'],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "timeswap",
+                                \'["raveena15, varunhawk19"]\') }}'
+    )
+}}
+
+SELECT
+  l.evt_tx_hash as transaction_hash,
+  l.evt_block_time as time,
+  l.isToken0 as token_0,
+  'Lend' as transaction_type,
```

Comment:
we don't have a sector-wide lending/borrowing spell yet -- which combines various projects across blockchains -- but in order to prep for that in the future, it may be good to follow standards from other sector spells. similar to column names, we typically will try to use lowercase on hardcode values as well